src/common/productGallery/interfaces/index.ts

```diff
@@ -0,0 +1,4 @@
+export interface IProductGalleryProps {
+  items: object[]; //Todo: change the type when we'll get json data
```

Comment:
In this case you can use any type for now since we don't know the type we will get from backend